core/server/lib/image/manipulator.js

```diff
@@ -8,15 +9,22 @@ const common = require('../common');
  * https://github.com/lovell/sharp/issues/1360.
  */
 const process = (options = {}) => {
-    let sharp, img;
+    let sharp, img, init, originalData, originalSize;
 
     try {
         sharp = require('sharp');
 
         // @NOTE: workaround for Windows as libvips keeps a reference to the input file
         //        which makes it impossible to fs.unlink() it on cleanup stage
         sharp.cache(false);
-        img = sharp(options.in);
+
+        init = fs.readFile(options.in)
+            .then((data) => {
+                originalData = data;
+
+                // @NOTE: have to user constructor with Buffer for sharp to be able to expose size property
```

Comment:
> have to user constructor

have to use ...

---

Just pointing this out, because i head to read this line 2 times to understand what you mean 🙊

Reply:
:see_no_evil: oh noooo!